packages/diagrams/frontend/sirius-components-diagrams/src/representation/DiagramRepresentation.tsx

```diff
@@ -949,6 +951,8 @@ export const DiagramRepresentation = ({
     [dispatch]
   );
 
+  const { enqueueSnackbar } = useContext(ToastContext).useToast();
```

Comment:
It's a strange API. I'd expect client code just to `useToast()` and not have to deal with notistack-specific details (`enqueueSnackbar`, the option object with a `variant` that requires explicit conversion from our own types).

Ideally, something like:
```js
const { addMessages } = useToast();
...
addMessages(messages); // Take a GQLMessage[] directly
```
and hide the `messages.map((message) => enqueueSnackbar(message.body, { variant: getVariantFromMessageLevel(message.level) }));` from our business code.

Reply:
I added a custom hook `useMultiToast`